jenkins-master/jobs/ondemand_update/config.xml

```diff
@@ -15,42 +15,31 @@
     </hudson.queueSorter.PrioritySorterJobProperty>
     <hudson.model.ParametersDefinitionProperty>
       <parameterDefinitions>
-        <hudson.model.StringParameterDefinition>
-          <name>APPLICATION</name>
-          <description>The application type.</description>
-          <defaultValue>firefox</defaultValue>
-        </hudson.model.StringParameterDefinition>
-        <hudson.model.StringParameterDefinition>
-          <name>BUILD_NUMBER</name>
-          <description>The build number of Firefox.</description>
-          <defaultValue>1</defaultValue>
-        </hudson.model.StringParameterDefinition>
         <hudson.model.ChoiceParameterDefinition>
-          <name>BUILD_TYPE</name>
-          <description>The build type of Firefox.</description>
+          <name>ALLOW_MAR_CHANNEL</name>
+          <description>MAR channel to be allowed for an update</description>
           <choices class="java.util.Arrays$ArrayList">
             <a class="string-array">
-              <string>release</string>
-              <string>candidate</string>
+              <string>None</string>
```

Comment:
It will allow testing of Firefox XYRC builds to Firefox XY+1 Beta builds. It's just a copy of the former mozmill driven job.